lib/delve.js

```diff
@@ -26,7 +26,19 @@ function addOutputMessage(messageType, message) {
 }
 
 function run(file, method) {
-	const { path: dlvPath, args } = store.getState().delve;
+	const { path: dlvPath, args, fileOverride } = store.getState().delve;
+
+	let trimmedFile = fileOverride.trim();
+	if (fileOverride && trimmedFile.trim !== "") {
+		const editor = atom.workspace.getActiveTextEditor();
+		const relativizedPath = atom.project.relativizePath(editor.getPath());
+		if (relativizedPath.length === 2) {
```

Comment:
Sounds good with me.  I wasn't aware that `atom.project.resolvePath` existed so that makes this much easier.